ts/src/utils.ts

```diff
@@ -175,10 +175,13 @@ export function parseSpec(specPath: string): RunSpecification {
         process.exit(1);
     }
 
-    // Add empty sanitizers array if the user didn't specify any
```

Comment:
Please refill the sanitizers. Im not interested in Sars-Covid-19.

Reply:
the virus is actually sars-cov-2, which causes the covid-19 respiratory infection